### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by SSEKPIUS**
In this project, we will refactor the Awesome books application to use JavaScript classes.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,98 +1,109 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-unused-vars */
```
Comment (mckent05) [inline index.js]: - I'll advise you to remove all disabled linters and try to fix the issues. Remember that linters help you to ensure that you write code that is clean and easy to maintain Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi guys,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ 
- Your code looks clean 👏 

## Required Changes ♻️
- Currently, the links in your project aren't displaying or switching to the right page when I Click on them. Please look into it and fix them 👍 

Hint: Use the class instance you initialized to call the method for toggling the page view 👍 👍 

Your Project:
awesoe books

_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (george827) [reply index.js]: hello @mckent05 , for the links switching to the right page section, this is the project requirement for the third section of this project. this is currently the second part which is refactor using classes. Hence for this project it's not part of the requirement

Comment (george827) [reply index.js]: @mckent05, This we are required to implement in the third project.

Comment (mckent05) [reply index.js]: Okay @george827, I noticed it was implemented in the previous milestone(PR), so I thought you introduced a bug in the code.

**Review by mckent05 — APPROVED** [review]
Hi team,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and good job! 🎉 

## Highlights
- Project works as expected 👏 
- All linter checks are passing correctly ✔️ 

## Optional suggestions
- Good job implementing the changes I suggested for toggling the view, though I saw your comment @george827, the links aren't working as expected because you aren't calling the methods correctly. Use `dmMethods.toggleView(index)` instead, and also consider calling `dmMethods.toggleView(index)` at the bottom of your code to ensure the right page loads when the DOM loads initially. 👍 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (george827) [reply index.js]: Hi @mckent05  sorry for the confusion 🙏. Thank you for the hint you gave.

**Pull request merged**